### PR TITLE
Fix :FAB doesn't close when cursor moves away

### LINKED ISSUE #1302: FAB doesn't close when focus is lost
fabnotclosing
The Floating Action button expands when hovered on, but doesn't collapse even when another button is clicked on. 
Expected behaviour: FAB collapses when another button is clicked, when the mouse is hovered over any other element, or when the action performed by the FAB is finished.

### PR DESCRIPTION
**Pull request by punwai**
Fixes #1302 

Fixed it so that it no longer stays up forever once the mouse moves 50px away from it.

demo

### CODE REVIEW

**Comment by punwai** [comment]
@robbyoconnor @muarachmann

**Comment by muarachmann** [comment]
@punwai please edit your title to something of the sort Fix :FAB doesn't close when focus is lost

File: interface/main/tabs/main.php [hunk 1]
```diff
@@ -361,19 +369,44 @@ function isEncounterLocked( encounterId ) {
...
+}, function(){
+    timer = setTimeout(function(){
+        $('#developer-mode').fadeOut(100);
+        $('#updater-options').fadeOut(100);
+        $('#updater-icon').css("border", "none");
+    }, 3000);
+});
+$('#updater-icon-wrapper').hover(function(){}, function(){
+    secondTimer = setTimeout(function(){
+        $('#developer-mode').fadeOut(100);
```
Comment (muarachmann) [inline interface/main/tabs/main.php]: excellent piece of jquery.... works pretty fine

File: interface/main/tabs/main.php [hunk 2]
```diff
@@ -361,19 +369,44 @@ function isEncounterLocked( encounterId ) {
...
+
 $('#updater-icon').click(function () {
     $('#updater-iframe').iziModal('open');
 });
 $('body').hover(function () {
     $('#developer-mode').css("display", "none");
     $('#updater-options').css("display", "none");
     $(this).css("border", "0px solid #000");
+}, function(){
+
```
Comment (muarachmann) [inline interface/main/tabs/main.php]: What this for i calls no method... nor does works

**Comment by muarachmann** [comment]
@Trodrige, @naveen17797 , @robbyoconnor  tested and it works like charm... Left for @punwai to answer some questions before i can give my approval.

File: interface/main/tabs/main.php (hunk 2 above)
Comment (punwai) [reply interface/main/tabs/main.php]: @muarachmann It's a mistake in my code, I'll remove it. I was meant to put it on another hover function but probably put it here instead. It doesn't do anything though so it won't break the code

Comment (punwai) [reply interface/main/tabs/main.php]: @muarachmann removed it!

File: interface/main/tabs/main.php (hunk 1 above)
Comment (aethelwulffe) [reply interface/main/tabs/main.php]: I never caught onto this happening before.  I think I should have seen something in the inspector.

Comment (aethelwulffe) [reply interface/main/tabs/main.php]: Wondering if we are setting developer mode by default for superusers ("administrators").

**Comment by aethelwulffe** [comment]
We have a Mua review, approval.  This tests, and giant horking icon fades away now instead of causing accidental upgrades!

**Pull request merged**